web/src/flow/stages/captcha/CaptchaStage.ts

```diff
@@ -157,10 +155,16 @@ export class CaptchaStage extends BaseStage<CaptchaChallenge, CaptchaChallengeRe
         if (this.captchaInteractive) {
             return html`${this.captchaContainer}`;
         }
+        if (this.embedded) {
+            return html``;
```

Comment:
**This *is* a change request.**

Please return `nothing` instead of an empty `html()` call.  Lit will create empty DOM-state tracking tags for an empty `html()`, which in the small is irrelevant, but it's still messy.

You can add `nothing` in the Lit base import:

```
import { CSSResult, PropertyValues, TemplateResult, html, nothing } from "lit";
```